opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/export/__init__.py

```diff
@@ -402,7 +402,7 @@ def get_metrics_data(
 
     def _receive_metrics(
         self,
-        metrics_data: "opentelemetry.sdk.metrics.export.MetricsData",
+        metrics_data: Optional[MetricsData],
```

Comment:
This would be a breaking change as `_receive_metrics` is "protected" and users are intended to override this abstractmethod

Reply:
Refactored